### PR TITLE
Loadtest

### PR DESCRIPTION
**Pull request by vcastellm**
# Description

This PR replaces Pandora's Box load testing with K6 using xk6-ethereum extension as specified in RFC-160

A couple of scenarios are initially provided, simple transfers and ERC20 transfers, take a look to `scenarios/` for details.

Extra output details that this method can provide are being reported to the slack channel and Datadog dashboards.

image

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [x] I have updated the official documentation
- [x] I have added sufficient documentation in code

## Testing

- [x] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

Run the test from local to devnet

### CODE REVIEW

File: scenarios/slack.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import http from "k6/http";
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
+export const payload = {
+  "blocks": [
+    {
+      "type": "header",
+      "text": {
+        "type": "plain_text",
+        "text": "Pandora's Box Results"
```
Comment (Stefan-Ethernal) [inline scenarios/slack.js]: Should we change the header text to something like:
Load Test Results or K6 Load Test Results?

File: .github/workflows/loadtest.yml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          uses: actions/checkout@v3
+        
+        - uses: datadog/agent-github-action@v1.3
+          with:
+            api_key: ${{ secrets.DD_API_KEY }}
+        
+        - name: Install xk6
+          run: |
+            go install go.k6.io/xk6/cmd/xk6@latest
+            xk6 build --with github.com/distribworks/xk6-ethereum@a3127e8173097e725da2890ee61782b1bea13fb7
```
Comment (Stefan-Ethernal) [inline .github/workflows/loadtest.yml:67]: Have you considered creating a release for xk6?

File: scenarios/multiple_ERC20.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    return {
+        accounts: fundTestAccounts(client, root_address),
+        contract_address: data.contract_address
+    };
+}
+
+let nonce = 0;
+let client;
+
+// VU client
```
Comment (Stefan-Ethernal) [inline scenarios/multiple_ERC20.js]: Just curious what does VU abbreviation stand for?

**Review by Stefan-Ethernal — COMMENTED** [review]
Wanted to propose different folders layout if it makes sense:
- loadtest
    - contracts (content unchanged)
    - helpers (content unchanged)
    - hooks (`slack.js` goes in here)
    - scenarios (`multiple.js` and `multiple_ERC20.js` go in here)

**Review by ferranbt — APPROVED** [review]
LGTM! I agree with @Stefan-Ethernal on the layout proposal.

File: .github/workflows/loadtest.yml (hunk 2 above)
Comment (vcastellm) [reply .github/workflows/loadtest.yml:67]: That is Blockspeed https://github.com/distribworks/blockspeed/releases

File: scenarios/slack.js (hunk 1 above)
Comment (vcastellm) [reply scenarios/slack.js]: Going to remove slack.js it's not used at this time

File: scenarios/multiple_ERC20.js (hunk 3 above)
Comment (vcastellm) [reply scenarios/multiple_ERC20.js]: Virtual User

**Comment by vcastellm** [comment]
@ferranbt @Stefan-Ethernal makes sense, changed folder layout

File: scenarios/slack.js (hunk 1 above)
Comment (Stefan-Ethernal) [reply scenarios/slack.js]: Just curious, how are we going to present the results?

**Review by Stefan-Ethernal — APPROVED** [review]
LGTM

File: scenarios/slack.js (hunk 1 above)
Comment (vcastellm) [reply scenarios/slack.js]: They are sent using `jq` + Slack GH Action trickery :)

**Pull request merged**